suction_control:
  - name: screw_tool_m3        # Name of screw fastening tool
    digital_in_port: 0         # The pin to which the pressure sensor is connected
    digital_out_port_vac: 0    # The pin to which the 'vac' of vacuum pump unit is connected
    digital_out_port_blow: 1   # The pin to which the 'blow' of vacuum pump unit is connected
  - name: screw_tool_m4
    digital_in_port: 1
    digital_out_port_vac: 2
    digital_out_port_blow: 3
  - name: screw_tool_m6
    digital_in_port: 2
    digital_out_port_vac: 4
    digital_out_port_blow: 5
  # - name: nut_tool_m6
  #   digital_in_port: 3
  #   digital_out_port_vac: 6
  #   digital_out_port_blow: 7
  - name: suction_tool
    digital_in_port: 3
    digital_out_port_vac: 6
    digital_out_port_blow: 7

